From 2d30b206719b384d37110d20722b2d73be585927 Mon Sep 17 00:00:00 2001
From: Xing Zheng <zhengxing@rock-chips.com>
Date: Fri, 1 Apr 2016 17:35:09 +0800
Subject: [PATCH] ARM64: rockchip_cros_defconfig: Add support DRM for cros

And removed:
----
-CONFIG_FB=y
-CONFIG_LCDC_RK3368=y
-CONFIG_LCDC_LITE_RK3X=y
-CONFIG_RK_IOMMU=y
-CONFIG_RK_IOVMM=y
----
which are unused on the chromeos.

Change-Id: Icd521b56b6285099d72d3bf25575466792b6d353
Signed-off-by: Xing Zheng <zhengxing@rock-chips.com>
---
 arch/arm64/configs/rockchip_cros_defconfig | 14 +++++++++-----
 1 file changed, 9 insertions(+), 5 deletions(-)

diff --git a/arch/arm64/configs/rockchip_cros_defconfig b/arch/arm64/configs/rockchip_cros_defconfig
index 1f2e08b5da31..6026fa6bc2f7 100644
--- a/arch/arm64/configs/rockchip_cros_defconfig
+++ b/arch/arm64/configs/rockchip_cros_defconfig
@@ -341,6 +341,13 @@ CONFIG_VIDEO_V4L2_SUBDEV_API=y
 CONFIG_MEDIA_USB_SUPPORT=y
 CONFIG_USB_VIDEO_CLASS=y
 CONFIG_V4L_PLATFORM_DRIVERS=y
+CONFIG_DRM=y
+CONFIG_DRM_LOAD_EDID_FIRMWARE=y
+CONFIG_DRM_ROCKCHIP=y
+CONFIG_ROCKCHIP_DW_HDMI=y
+CONFIG_ROCKCHIP_DW_MIPI_DSI=y
+CONFIG_ROCKCHIP_ANALOGIX_DP=y
+CONFIG_DRM_PANEL_SIMPLE=y
 CONFIG_POWERVR_ROGUE_M=y
 CONFIG_MALI_MIDGARD=y
 CONFIG_MALI_MIDGARD_DEBUG_SYS=y
@@ -350,14 +357,13 @@ CONFIG_MALI_DEBUG_SHADER_SPLIT_FS=y
 CONFIG_MALI_PLATFORM_THIRDPARTY=y
 CONFIG_MALI_PLATFORM_THIRDPARTY_NAME="rk"
 CONFIG_MALI_DEBUG=y
-CONFIG_FB=y
+
 CONFIG_BACKLIGHT_LCD_SUPPORT=y
 # CONFIG_LCD_CLASS_DEVICE is not set
 # CONFIG_BACKLIGHT_GENERIC is not set
 CONFIG_BACKLIGHT_PWM=y
 CONFIG_FB_ROCKCHIP=y
-CONFIG_LCDC_RK3368=y
-CONFIG_LCDC_LITE_RK3X=y
+
 CONFIG_LCD_MIPI=y
 CONFIG_RK_TRSM=y
 CONFIG_RK31XX_LVDS=y
@@ -522,8 +528,6 @@ CONFIG_COMMON_CLK_RK808=y
 # CONFIG_COMMON_CLK_XGENE is not set
 CONFIG_MAILBOX=y
 CONFIG_ROCKCHIP_IOMMU=y
-CONFIG_RK_IOMMU=y
-CONFIG_RK_IOVMM=y
 CONFIG_ROCKCHIP_PM_DOMAINS=y
 CONFIG_PM_DEVFREQ=y
 CONFIG_DEVFREQ_GOV_SIMPLE_ONDEMAND=y
-- 
2.35.3

